recipes/calliope/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "calliope" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "78c483e996d3f51f6285a505a82df002ac3079f33894563b2c5ec08dea265025" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<35]
+  entry_points:
+    - {{ name }} = {{ name }}.cli:cli
+
+requirements:
+  build:
```

Comment:
I believe that the `--single-version-externally-managed` flag in the build script makes it such that the `install_requires` in the `setup.py` is ignored. Therefore, please limit the packages listed in this section to those that are actually required to build the recipe.